src/renderer/pages/CreateNewProjectPage.vue

```diff
@@ -1,32 +1,24 @@
 <template id="CreateNewProject">
     <div>
-        <h1 class="page-title" style="margin-top: 10vh; margin-bottom: 5vh;">VIDEO BONES</h1>
+        <h1 class="page-title" style="margin-bottom: 5vh; margin-top: 10vh;">VIDEO BONES</h1>
 
         <menu class="options-menu" style="margin: auto;">
 
             <div style="grid-column: 1 / 3; grid-row: 1 / 2;" class="named-input-container">
                 <h3>Project Name</h3>
-                <input id="project-name-input" placeholder="Ex: Gangnam Style A Capella" type="text">
+                <input @input="updateLocation" id="project-name-input" placeholder="Ex: Gangnam Style A Capella" type="text">
```

Comment:
```suggestion
                <input @input="updateLocation()" id="project-name-input" placeholder="Ex: Gangnam Style A Capella" type="text">
```
Good to have the brackets to show it's a function. ok !! ok!